### ISSUE TITLE
progress-stepper: missing aria-labels for some states

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#1614]
I missed this in the PR, but we are missing aria-labels on the icons in some of the steps (e.g. the "Complete" steps/icons). The states are available in Figma.

We'd need to make sure these are present on coreui side too.

![Screen Shot 2021-12-10 at 1 02 47 PM](https://user-images.githubusercontent.com/38065/145641035-f3d00ab1-ad05-401e-b376-f9ba4251aa66.png)
